core/tests/performance_framework/perf_services.py

```diff
@@ -58,21 +60,64 @@ class SeleniumPerformanceDataFetcher(object):
     # XHR requests made post initial page load will not be recorded.
     WAIT_DURATION_SECS = 3
 
-    def __init__(self, browser=DEFAULT_BROWSER_SOURCE):
+    BASE_URL = 'http://localhost:%d' % feconf.PERFORMANCE_TESTS_SERVER_PORT
+
+    LIBRARY_URL_SUFFIX = '/library'
+    EDITOR_URL_SLICE = '/create/'
+    PLAYER_URL_SLICE = '/explore/'
+    PROFILE_URL_SLICE = '/profile/'
+    LOGIN_URL_SUFFIX = '/_ah/login'
+    ADMIN_URL_SUFFIX = '/admin'
+    DASHBOARD_URL_SUFFIX = '/dashboard'
+    SPLASH_URL_SUFFIX = '/splash'
+    SIGNUP_URL_SUFFIX = '/signup'
+    DUMMY_URL = '/robots.txt'
+
+    def __init__(self, browser=DEFAULT_BROWSER_SOURCE, preload_options=None):
+        """preload_options:
```

Comment:
Is this a dict? If so, say so.

That said, why not pass each argument individually? That would probably be clearer.